cpp/src/io/orc/stats_enc.cu

```diff
@@ -27,6 +27,8 @@ namespace cudf::io::orc::gpu {
 
 using strings::detail::fixed_point_string_size;
 
+constexpr bool enable_nanosecond_statistics = false;
```

Comment:
Could we link https://github.com/rapidsai/cudf/issues/14325 here as this flag is what needs to be be flipped to add the currently excluded statistic?